README.md

```diff
@@ -17,11 +19,12 @@ CloudNativePG was originally built and sponsored by [EDB](https://www.enterprise
 - [Code of conduct](CODE_OF_CONDUCT.md)
 - [Governance policies](GOVERNANCE.md)
 - [Contributing](CONTRIBUTING.md)
+- [Adopters](ADOPTERS.md)
 - [License](LICENSE)
 
 ## Getting Started
 
-The best way to get started is with the ["Quickstart"](docs/src/quickstart.md)
+The best way to get started is with the ["Quickstart"](https://cloudnative-pg.io/documentation/current/quickstart/)
```

Comment:
I think it's better to keep links local. Both in GH and in local checkouts, does what we want, and the link could mask mistakes.

Reply:
Ok!